code/modules/research/designs/weapon.dm

```diff
@@ -191,37 +191,37 @@
 	name = "Auto .35, laser"
 	desc = "Auto .35 cases fitted with a one time use laser payload."
 	build_path = /obj/item/ammo_magazine/ammobox/pistol_35/laser
-	materials = list(MATERIAL_STEEL = 5, MATERIAL_SILVER = 15, MATERIAL_GOLD = 15)
```

Comment:
Why are you removing the silver?
We aggreed on lower the costs? Not replacing the silver

Reply:
Ah NVM I see what you did, you made it more constant with .223